doc/classes/TextEdit.xml

```diff
@@ -417,6 +417,13 @@
 				Returns the width in pixels of the [param wrap_index] on [param line].
 			</description>
 		</method>
+		<method name="get_line_with_ime" qualifiers="const">
+			<return type="String" />
+			<param index="0" name="line" type="int" />
+			<description>
+				Returns line text including current IME preedit string.
```

Comment:
I am... a bit confused on what this is, description doesn't help.

Reply:
`preedit string` is pretty standard terminology for IME. It's part of the text which is currently edited (controlled by IME driver), but it's not part of the `text` property until the edit is committed. So this function returns text which is currently displayed.